flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBOperationUtils.java

```diff
@@ -119,14 +122,18 @@ public static void registerKvStateInformation(
      *
      * <p>Creates the column family for the state. Sets TTL compaction filter if {@code
      * ttlCompactFiltersManager} is not {@code null}.
+     *
+     * @param importFilesMetaData if not empty, we import the files specified in the metadata to the
+     *     column family.
      */
     public static RocksDBKeyedStateBackend.RocksDbKvStateInfo createStateInfo(
             RegisteredStateMetaInfoBase metaInfoBase,
             RocksDB db,
             Function<String, ColumnFamilyOptions> columnFamilyOptionsFactory,
             @Nullable RocksDbTtlCompactFiltersManager ttlCompactFiltersManager,
             @Nullable Long writeBufferManagerCapacity,
-            ICloseableRegistry cancelStreamRegistryForRestore) {
+            ICloseableRegistry cancelStreamRegistryForRestore,
+            List<ExportImportFilesMetaData> importFilesMetaData) {
```

Comment:
nit: please change the order of parameters (first the list, then ICloseableRegistry) so that it's consistent with the order of `createColumnFamily`.